frontend/src/components/Account/common/ProfileDetails/ProfileDetails.js

```diff
@@ -1,46 +1,23 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
-import "./ProfileDetails.scss";
 
-const renderField = (field = {}) => {
-  let processedValue = field.value;
-  if (typeof field.value === "boolean") {
-    processedValue = field.value ? "Da" : "Nu";
-  }
-  return (
-    field.label && (
-      <div className="profile__field" key={field.label}>
-        <strong> {field.label}:</strong> {processedValue}
-      </div>
-    )
-  );
-};
+import { mapProfileDetails } from "./ProfileDetails.constants";
 
-const renderFields = fields => {
-  return fields.map(renderField);
-};
+import "./ProfileDetails.scss";
 
-const ProfileDetails = ({ fields, isSelf }) => {
-  const leftColumnFields = [
-    fields.name,
-    fields.relationship,
-    fields.phoneNumber,
-    fields.location,
-    fields.age,
-    fields.gender
-  ];
-  const rightColumnFields = [
-    fields.smoking,
-    fields.comorbidities,
-    fields.livingAlone,
-    fields.inIsolation,
-    fields.othersInHouseholdIsolation
-  ];
-  if (!fields || !Object.keys(fields).length) {
+const ProfileDetails = ({
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Alternatively, you could place an underscore in front of the variable that is unused